src/eventSchemas/models/schemaCodeLangs.ts

```diff
@@ -3,7 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { Runtime } from 'aws-sdk/clients/lambda'
 import { Set } from 'immutable'
+import { supportsEventBridgeTemplates } from '../../../src/lambda/models/samTemplates'
```

Comment:
`supportsEventBridgeTemplates` feels like it is more of a concern of schemaCodeLans than sam templates. What do you think about moving it into this file?

Reply:
Sure will do that as well.